index.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Apple.com-clone-project</title>
+    <link rel="stylesheet" type="text/css" href="./stylesheets/reset.css" />
+    <link rel="stylesheet" type="text/css" href="./stylesheets/style.css" />
+  </head>
+  <body>
+    <section class="main">
+      <header class="navlist">
+        <div>
+          <ul>
+            <li class="nav-item bar-item1">
+              <a href="#"><i class="fab fa-apple"></i></a>
+            </li>
+            <li class="nav-item"><a href="#">Store</a></li>
+            <li class="nav-item"><a href="#">Mac</a></li>
+            <li class="nav-item"><a href="#">iPod</a></li>
+            <li class="nav-item"><a href="#">iPhone</a></li>
+            <li class="nav-item"><a href="#">iPad</a></li>
+            <li class="nav-item"><a href="#">iTunes</a></li>
+            <li class="nav-item"><a href="#">Support</a></li>
+            <li class="nav-item bar-item2" id="search">
+              <a href="#"><i class="fa fa-search"></i></a>
+              <input type="text" placeholder="" />
+            </li>
+          </ul>
+        </div>
+      </header>
+
+      <div class="page-text">
+        <img
+          class="small"
+          src="https://web.archive.org/web/20140228164946im_/http://images.apple.com/home/images/your_verse_title_small.png"
+          alt="What will your verse be?"
+          width="980"
+          height="104"
+        />
+        <img
+          class="small"
+          src="https://web.archive.org/web/20140228164946im_/http://images.apple.com/home/images/your_verse_subhead_small.png"
+          alt="Everyone has something to share. See how people are using iPad in their unique ways to add to the world’s story."
+          width="980"
+          height="82"
+        />
+        <h3>
+          <a href="#"
+            >Watch the "Your Verse" and explore the stories<i
+              class="far fa-play-circle"
+            ></i
+          ></a>
+        </h3>
+      </div>
+    </section>
+
+    <div class="apple-images">
+      <div><img src="./images/iPhone-image1.jpg" alt="iPhone-advert" /></div>
+      <div><img src="./images/iPhone-image2.jpg" alt="iPhone-advert" /></div>
+      <div><img src="./images/iPhone-image3.jpg" alt="iPhone-advert" /></div>
+      <div><img src="./images/iPhone-image4.jpg" alt="iPhone-advert" /></div>
```

Comment:
As this project is background and gradients based please remove these images and make them background images instead.